eq-author-api/schema/tests/questionnaire.test.js

```diff
@@ -1,9 +1,10 @@
 const {
   buildQuestionnaire,
 } = require("../../tests/utils/questionnaireBuilder");
-
+const executeQuery = require("../../tests/utils/executeQuery");
```

Comment:
I guess this and the mutation below are not needed.

Reply:
This and the mutations both get used in the tests as I tend to use them to fire  off the mutations rather than the test functions we use in build questionnaire function. Might be worth discussing at some point which method is better and if the (test)? functions are in the correct place?

Reply:
Yeah, lets leave it for the moment then. We can discuss once `797` is done.